voxel51/users/api.py

```diff
@@ -473,26 +487,41 @@ def delete_data(self, data_id):
         res = self._requests.delete(endpoint, headers=self._header)
         _validate_response(res)
 
-    def batch_update_data_ttl(self, data_ids, days):
-        '''Updates the expiration date of the data with the given IDs.
+    def batch_update_data_ttl(self, data_ids, days=None, expiration_date=None):
+        '''Updates the expiration dates of the data with the given IDs.
+
+        Note that if the expiration date of the data after modification is in
+        the past, the data will be deleted.
 
-        To decrease the lifespan of the data, provide a negative number. Note
-        that if the expiration date of data after modification is in the past,
-        the data will be deleted.
+        Exactly one keyword argument must be provided.
 
         Args:
             data_ids (list): the data IDs
-            days (float): the number of days by which to extend the lifespan
-                of the data
+            days (float, optional): the number of days by which to extend the
+                lifespan of the data. To decrease the lifespan of the data,
+                provide a negative number
+            expiration_date (datetime|str, optional): a new expiration date for
+                the data. If a string is provided, it must be in ISO 8601
+                format, e.g., "YYYY-MM-DDThh:mm:ss.sssZ". If a non-UTC timezone
+                is included in the datetime or string, it will be respected
 
         Returns:
-            a dictionary mapping data IDs to True/False values indicating
-                whether the data was successfully processed (True)
+            a dictionary mapping data IDs to dictionaries indicating whether
```

Comment:
@lethosor thanks for fixing the documentation for the actual return structures here